source/simulator/parameters.cc

```diff
@@ -269,77 +269,6 @@ namespace aspect
-                       "of the Stokes system is computed. This approximate $A$ is used in the "
-                       "preconditioning used in the GMRES solver. The exact definition of this "
-                       "block preconditioner for the Stokes equation can be found in "
-                       "\\cite{KHB12}.");
-
-    prm.declare_entry ("Linear solver S block tolerance", "1e-6",
-                       Patterns::Double(0,1),
-                       "A relative tolerance up to which the approximate inverse of the $S$ block "
-                       "(i.e., the Schur complement matrix $S = BA^{-1}B^{T}$) of the Stokes "
-                       "system is computed. This approximate inverse of the $S$ block is used "
-                       "in the preconditioning used in the GMRES solver. The exact definition of "
-                       "this block preconditioner for the Stokes equation can be found in "
-                       "\\cite{KHB12}.");
-
-    prm.declare_entry ("Number of cheap Stokes solver steps", "200",
-                       Patterns::Integer(0),
-                       "As explained in the paper that describes ASPECT (Kronbichler, Heister, and Bangerth, "
-                       "2012, see \\cite{KHB12}) we first try to solve the Stokes system in every "
-                       "time step using a GMRES iteration with a poor but cheap "
-                       "preconditioner. By default, we try whether we can converge the GMRES "
-                       "solver in 200 such iterations before deciding that we need a better "
-                       "preconditioner. This is sufficient for simple problems with variable "
-                       "viscosity and we never need the second phase with the more expensive "
-                       "preconditioner. On the other hand, for more complex problems, and in "
-                       "particular for problems with strongly nonlinear viscosity, the 200 "
-                       "cheap iterations don't actually do very much good and one might skip "
-                       "this part right away. In that case, this parameter can be set to "
-                       "zero, i.e., we immediately start with the better but more expensive "
-                       "preconditioner.");
-
-    prm.declare_entry ("Maximum number of expensive Stokes solver steps", "1000",
-                       Patterns::Integer(0),
-                       "This sets the maximum number of iterations used in the expensive Stokes solver. "
-                       "If this value is set too low for the size of the problem, the Stokes solver will "
-                       "not converge and return an error message pointing out that the user didn't allow "
-                       "a sufficiently large number of iterations for the iterative solver to converge.");
-
     prm.declare_entry ("Temperature solver tolerance", "1e-12",
```

Comment:
I thought we had agreed to also move the temperature and composition solver tolerance if we move the linear solver tolerance? 
I am not insisting on this (and the last time we talked about this, I had assumed none of these tolerances would be moved, that is why I argued they should stay where they are), but I just think this can be quite confusing. Why is the Linear solver tolerance in its own subsection, while the temperature and composition solver tolerances aren't? 
We can also do that in a follow-up pull request, but we should at least discuss again what the final state for the release should be.